app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -218,10 +218,82 @@
 }
 
 .boundary-line {
+  border-bottom: 2px solid #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.